### PR TITLE
[onert] Fix StaticTensorManager to use the only NativeTensors on cpu

### PR DESCRIPTION
**Pull request by YongseopKim**
Fix cpu::StaticTensorManager to use the only NativeTensors

ONE-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

**Comment by YongseopKim** [comment]
Fix https://github.com/Samsung/ONE/pull/3229#issuecomment-664098155

**Comment by YongseopKim** [comment]
for #3039
draft #3037

**Comment by YongseopKim** [comment]
PTAL @wateret @chunseoklee and @periannath

File: runtime/onert/backend/cpu/StaticTensorManager.cc [hunk 1]
```diff
@@ -97,8 +97,6 @@ void StaticTensorManager::iterate(const std::function<void(const ir::OperandInde
 {
   for (const auto &it : _tensors->managed_tensors())
```
Comment (periannath) [inline runtime/onert/backend/cpu/StaticTensorManager.cc:98]: This line is not part of this PR, but it seems that this line should be `_tensors->native_tensors()` since `MigrantTensor` becomes `NativeTensor`.

Comment (wateret) [reply runtime/onert/backend/cpu/StaticTensorManager.cc:98]: I'll do that, thanks.
